dangerzone/util.py

```diff
@@ -45,6 +46,26 @@ def get_resource_path(filename: str) -> str:
     return str(resource_path)
 
 
+def get_tessdata_dir() -> str:
+    if (
+        getattr(sys, "dangerzone_dev", False)
+        or platform.system() == "Windows"
```

Comment:
```suggestion
        or platform.system() in ("Windows", "Darwin")
        ```

Reply:
Fixed in 4c7db48c598d575f33a63c139c721b.